packages/trpc/server/routers/viewer/slots/util.test.ts

```diff
@@ -1,90 +0,0 @@
-import { describe, expect, it } from "vitest";
```

Comment:
`getUsersWithCredentialsConsideringContactOwner` was removed because all this logic is already handled in `findQualifiedUsers`